pkg/kubelet/kubelet.go

```diff
@@ -504,7 +504,8 @@ func NewMainKubelet(
 		hostname,
 		klet.podManager,
 		klet.kubeClient,
-		klet.volumePluginMgr)
+		klet.volumePluginMgr,
+		klet.containerRuntime)
```

Comment:
Retract my comment since this would slightly increase the window for race condition to happen.